src/system/Dialog/Dialog.js

```diff
@@ -17,6 +17,10 @@ const Dialog = ( { trigger, position = 'left', startOpen = false, content, disab
 	const dialogRef = useRef( null );
 
 	const closeDialog = e => {
+		if ( props.variant !== 'modal' ) {
```

Comment:
This breaks the sidebar... if you click in the input, it closes the sidebar.
Maybe it will work if we check that the variant was not defined.

https://user-images.githubusercontent.com/818725/152016301-6ce6ae75-ba17-415e-bd37-d99ecc271c0c.mp4

Reply:
Good catch!

Thanks for the review, @robersongomes !